SPECS/liburing/liburing.spec

```diff
@@ -1,16 +1,12 @@
 Summary:        Linux-native io_uring I/O access library
 Name:           liburing
-Version:        2.0
-Release:        3%{?dist}
+Version:        2.4
```

Comment:
Curious if we can move up to v2.5 - https://git.kernel.dk/cgit/liburing/tag/?h=liburing-2.5 / https://brick.kernel.dk/snaps/liburing-2.5.tar.gz